docs/pt/docs/tutorial/first-steps.md

```diff
@@ -0,0 +1,333 @@
+* `GET`: para ler dados.
+* `PUT`: para atualizar dados.
+* `DELETE`: para deletar dados.
+
+Portanto, no OpenAPI, cada um dos métodos HTTP é chamado de "operação".
+
+Vamos chamá-los de "**operações**" também.
+
+#### Defina um *decorador de rota*
+
+```Python hl_lines="6"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+O `@app.get("/")` diz ao **FastAPI** que a função logo abaixo é responsável por tratar as requisições que vão para:
+
+* a rota `/`
+* usando o  <abbr title="o método HTTP GET">operador <code>get</code></abbr>
+
+!!! info "`@decorador`"
+    Essa sintaxe `@alguma_coisa` em Python é chamada de "decorador".
+
+    Você o coloca em cima de uma função. Como um chapéu decorativo (acho que é daí que vem o termo).
+
+    Um "decorador" pega a função abaixo e faz algo com ela.
+
+    Em nosso caso, este decorador informa ao **FastAPI** que a função abaixo corresponde a **rota** `/` com uma **operação** `get`.
+
+    É o "**decorador de rota**".
+
+Você também pode usar as outras operações:
+
+* `@app.post()`
+* `@app.put()`
+* `@app.delete()`
+
+E os mais exóticos:
+
+* `@app.options()`
+* `@app.head()`
+* `@app.patch()`
+* `@app.trace()`
+
+!!! tip "Dica"
+    Você está livre para usar cada operação (método HTTP) como desejar.
+
+    O **FastAPI** não impõe nenhum significado específico.
+
+    As informações aqui são apresentadas como uma orientação, não uma exigência.
+
+    Por exemplo, ao usar GraphQL, você normalmente executa todas as ações usando apenas operações `POST`.
+
+### Passo 4: defina uma **função de rota**
+
+Esta é a nossa "**função de rota**":
+
+* **rota**: é `/`.
+* **operação**: é `get`.
+* **função**: é a função abaixo do "decorador" (abaixo do `@app.get("/")`).
+
+```Python hl_lines="7"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+Esta é uma função Python.
+
+Ela será chamada pelo **FastAPI** sempre que receber uma requisição para a URL "`/ `" usando uma operação `GET`.
+
+Neste caso, é uma função `assíncrona`.
+
+---
+
+Você também pode defini-la como uma função normal em vez de `async def`:
+
+```Python hl_lines="7"
+{!../../../docs_src/first_steps/tutorial003.py!}
+```
+
+!!! nota
+    Se você não sabe a diferença, verifique o [Async: *"Com pressa?"*](../async.md#com-pressa){.internal-link target=_blank}.
+
+### Passo 5: retorne o conteúdo
+
+```Python hl_lines="8"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+Você pode retornar um `dict`, `list` e valores singulares como `str`, `int`, etc.
+
+Você também pode devolver modelos Pydantic (você verá mais sobre isso mais tarde).
+
+Existem muitos outros objetos e modelos que serão convertidos automaticamente para JSON (incluindo ORMs, etc). Tente usar seus favoritos, é altamente provável que já sejam compatíveis.
+
+## Recapitulando
+
+* Importe `FastAPI`.
+* Crie uma instância do `app`.
+* Escreva um **decorador de rota** (como `@app.get("/")`).
+* Escreva uma **função de rota** (como `def root(): ...` acima).
```

Comment:
I think there's an error with the original doc in English. As far as I'm concerned the correct word should be  "below", not "above" since the full example could be:

```
@app.get("/")
def root(): 
    ...
```
So, the full line would be:
```suggestion
* Escreva uma **função para a operação da rota** (como `def root(): ...`) abaixo.
```
What do you think? Am I missing something?